docs/buildah-pull.md

```diff
@@ -0,0 +1,112 @@
+# buildah-pull "1" "July 2018" "buildah"
+
+## NAME
+buildah\-pull - Creates a new working container, either from scratch or using a specified image as a starting point.
+
+## SYNOPSIS
+**buildah pull** [*options*] *image*
+
+## DESCRIPTION
+Pulls an image based upon the specified image name.  Image names
+use a "transport":"details" format.
+
+Multiple transports are supported:
+
+  **dir:**_path_
+  An existing local directory _path_ containing the manifest, layer tarballs, and signatures in individual files. This is a non-standardized format, primarily useful for debugging or noninvasive image inspection.
+
+  **docker://**_docker-reference_ (Default)
+  An image in a registry implementing the "Docker Registry HTTP API V2". By default, uses the authorization state in `$XDG\_RUNTIME\_DIR/containers/auth.json`, which is set using `(podman login)`. If the authorization state is not found there, `$HOME/.docker/config.json` is checked, which is set using `(docker login)`.
+  If _docker-reference_ does not include a registry name, *localhost* will be consulted first, followed by any registries named in the registries configuration.
+
+  **docker-archive:**_path_
+  An image is retrieved as a `docker load` formatted file.
+
+  **docker-daemon:**_docker-reference_
+  An image _docker-reference_ stored in the docker daemon's internal storage.  _docker-reference_ must include either a tag or a digest.  Alternatively, when reading images, the format can also be docker-daemon:algo:digest (an image ID).
+
+  **oci-archive:**_path_**:**_tag_
+  An image _tag_ in a directory compliant with "Open Container Image Layout Specification" at _path_.
+
+  **ostree:**_image_[**@**_/absolute/repo/path_]
+  An image in local OSTree repository.  _/absolute/repo/path_ defaults to _/ostree/repo_.
+
+### DEPENDENCIES
+
+Buildah resolves the path to the registry to pull from by using the /etc/containers/registries.conf
+file, registries.conf(5).  If the `buildah pull` command fails with an "image not known" error,
+first verify that the registries.conf file is installed and configured appropriately.
+
+## RETURN VALUE
+The image ID of the image that was pulled.  On error, 1 is returned and errno is returned.
```

Comment:
[...] and errno is **set**?

Reply:
This seems like a function message. Does errno mean anything to normal users.
On error the non zero errno is set?

Reply:
I cut/pasted this from the 'buildah from' man page.  I'll change the last sentence in both to just  "On error 1 is returned."